tests/network/vmi_networking.go

```diff
@@ -857,36 +868,24 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				}
 			})
 
-			DescribeTable("[Conformance] preserves connectivity", func(ipFamily k8sv1.IPFamily, ports []v1.Port) {
-				libnet.SkipWhenClusterNotSupportIpFamily(virtClient, ipFamily)
+			DescribeTable("[Conformance] preserves connectivity [IPv4]", func(ports []v1.Port) {
```

Comment:
Better not use `[]` as these are used for labeling the tests with keywords (that are later used for filtering).
Or maybe it was your explicit intention?

Reply:
Yes, that was the intention

Reply:
OK, so who is using this label now?

Reply:
[This](https://github.com/kubevirt/kubevirt/blob/f5973335f4fb4f504c0513a42cb54841528bdb4d/automation/test.sh#L365) is how a label is used. As you are not using it for that purpose, I think you should drop the parenthesis.